pkg/cleaning/cleanup.go

```diff
@@ -887,6 +887,13 @@ You should specify a token with delete:packages and read:packages scopes to remo
 Check --repo-github-token option.
 Be aware that the token provided to GitHub Actions workflow is not enough to remove package versions.
 Read more details here https://werf.io/documentation/v1.2/advanced/supported_container_registries.html#github-packages`, err)
+	case docker_registry.IsSelectelUnauthorizedErr(err):
+		return fmt.Errorf(`%w
+
+You should specify Serectel cloud container registry (cr) credentials: username, password, account and VPC to remove tags with Selectel CR API.
+Check --repo-selectel-username, --repo-selectel-password, --repo-selectel-account and --repo-selectel-vpc or --repo-selectel-vpc-id options.
+Be aware that access to the resource is forbidden with personal access token.
+Read more details here https://werf.io/documentation/v1.2/advanced/supported_container_registries.html#selectel-cr`, err)
```

Comment:
There is no such section in the documentation.
I am not sure that this part is related to selectel container registry too.

Reply:
I will add this section